chatgpt-shell.el

```diff
@@ -1418,15 +1405,22 @@ enables additional key bindings.
           (chatgpt-shell--primary-buffer)
         (chatgpt-shell--put-source-block-overlays)))))
 
+(defun chatgpt-shell-ask-chatgpt-region ()
```

Comment:
Let's remove this function. Looks like it's missing some parts and we already have functions that do a similar thing:

`chatgpt-shell-send-region`
`chatgpt-shell-send-and-review-region`